### PR TITLE
dos2unix: add page

### PR DESCRIPTION
**Pull request by sarav-egnaroinc**
man page for dos2unix - DOS/MAC to UNIX text file format converter

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [ ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/dos2unix.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+> DOS/Mac to UNIX and vice versa text file format converter.
+
+- Convert DOS file format to UNIX format:
+
+`dos2unix {{filename}}`
+
+- Convert DOS file format to UNIX format:
+
+`unix2dos {{filename}}`
```
Comment (agnivade) [inline pages/linux/dos2unix.md]: Let us not put 2 different commands in the same page. You can add a subheading at the top mentioning "Also see `unix2dos`"

File: pages/linux/dos2unix.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# dos2unix
+
+> DOS/Mac to UNIX and vice versa text file format converter.
+
+- Convert DOS file format to UNIX format:
+
+`dos2unix {{filename}}`
```
[the remark this answers is not in the document]
Comment (sarav-egnaroinc) [reply pages/linux/dos2unix.md]: updated

File: pages/linux/dos2unix.md (hunk 1 above)
Comment (sarav-egnaroinc) [reply pages/linux/dos2unix.md]: added

File: pages/linux/dos2unix.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# dos2unix
+
+> DOS/Mac to UNIX and vice versa text file format converter.
+> Also check unix2dos
+
+- Convert a file in the DOS format to the UNIX format:
+
+`dos2unix {{path/to/file}}`
+
+- Convert and replace a file while keeping original date stamp:
```
Comment (agnivade) [inline pages/linux/dos2unix.md]: I think timestamp is more common than "date stamp" ?

File: pages/linux/dos2unix.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`dos2unix -k {{filename}}`
```
Comment (agnivade) [inline pages/linux/dos2unix.md]: Please use a consistent token style. Above, you have used path/to/file, here filename. Let us just use path/to/file everywhere. For cases of input and output file, have path/to/input_file and path/to/output_file

File: pages/linux/dos2unix.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Convert from DOS to UNIX format and write to a newfile:
```
Comment (agnivade) [inline pages/linux/dos2unix.md]: new file

File: pages/linux/dos2unix.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`dos2unix -k {{input filename}} {{output filename}}`
+
+- Suppress all warning and messages while conversion:
```
Comment (agnivade) [inline pages/linux/dos2unix.md]: Did you mean to say "all warning messages" ?

Comment (waldyrious) [reply pages/linux/dos2unix.md]: Also please change "while" --> "during".

**Comment by mebeim** [comment]
Should we just take care of this ourselves? It's been > 1 month.

**Comment by agnivade** [comment]
If the changes are trivial, we usually take care of it ourselves. In this case, I think there are quite a few changes.

**Pull request closed without merging**